src/main/java/com/demo/classroom/Entity/Book.java

```diff
@@ -0,0 +1,29 @@
+package com.demo.classroom.Entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long bookId;
+
+    private String name;
+    private String author;
+
+    @ManyToOne
+    @JoinColumn(name = "course_id")
```

Comment:
Can it take null value?

Reply:
No, I can't take null values as a book must be suggested to a particular course. It is solved now

Reply:
then we should make it non-nullable.